templates/preview/local-env.sh

```diff
@@ -0,0 +1,11 @@
+export TIDEPOOL_HYDROPHONE_SERVICE="{
```

Comment:
do we need 2 environment files?

Reply:
we need the local-env.sh for local dev
The other environment file contain what is needed when it is deployed. It is not mandatory as we use ansible to generate a configuration file anyway...
Is it a problem to ship the package with a default env file?

Reply:
I thought we usually have 1 env file that we override based on context (local dev, deployment here or there..)